src/Actions/StubResolver/CopyStubAction.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace KoalaFacade\DiamondConsole\Actions\StubResolver;
+
+use Illuminate\Filesystem\Filesystem;
+use Illuminate\Support\Str;
+use KoalaFacade\DiamondConsole\Foundation\Action;
+
+class CopyStubAction extends Action
+{
+    /**
+     * Copy Stub to Diamond Infrastructure
+     *
+     * @param  string  $stubPath
+     * @param  string  $destinationPath
+     * @param  string  $name
+     * @param  array<string>  $replacements
+     * @return void
+     */
+    public function execute($stubPath, $destinationPath, $name, $replacements): void
+    {
+        $filesystem = new Filesystem;
+        $filePath = $destinationPath . '/' . $name . '.php';
```

Comment:
can move $filepath variable to top before $filesystem, like it strage when try to read the flow